app/channels/submission/test_runs_channel.rb

```diff
@@ -1,6 +1,6 @@
 class Submission::TestRunsChannel < ApplicationCable::Channel
   def subscribed
-    submission = current_user.solutions.find_by!(uuid: params[:submission_uuid])
+    submission = current_user.submissions.find_by!(uuid: params[:submission_uuid])
     stream_for submission
   end
```

Comment:
Let's remove this file and use the real Channel.

Reply:
This is the real channel.

Reply:
:D